addons/l10n_ar/demo/account_tax_template_demo.xml

```diff
@@ -27,4 +27,96 @@
         <field eval="0.1" name="amount"/>
     </record>
 
+    <record id="ri_tax_other_taxes_ventas" model="account.tax">
+        <field name="name">Other Tax</field>
+        <field name="description">Other Tax</field>
+        <field name="sequence">4</field>
+        <field name="active" eval="False"/>
+        <field name="amount_type">fixed</field>
+        <field name="amount" eval="0.1"/>
+        <field name="invoice_repartition_line_ids" model="account.account" eval="[(5, 0, 0),
+            (0,0, {
+                'factor_percent': 100,
+                'repartition_type': 'base',
+            }),
+
+            (0,0, {
+                'factor_percent': 100,
+                'repartition_type': 'tax',
+                'account_id': obj().search([
+                    ('company_id', '=', obj().env.ref('l10n_ar.company_ri').id),
+                    ('group_id', '=', obj().env.ref('l10n_ar.account_group_otros_impuestos').id)
+                ], limit=1).id,
+            }),
+        ]"/>
+        <field name="refund_repartition_line_ids" model="account.account" eval="[(5, 0, 0),
+            (0,0, {
+                'factor_percent': 100,
+                'repartition_type': 'base',
+            }),
+
+            (0,0, {
+                'factor_percent': 100,
+                'repartition_type': 'tax',
+                'account_id': obj().search([
+                    ('company_id', '=', obj().env.ref('l10n_ar.company_ri').id),
+                    ('group_id', '=', obj().env.ref('l10n_ar.account_group_otros_impuestos').id)
+                ], limit=1).id,
+            }),
+        ]"/>
+        <field name="type_tax_use">sale</field>
+        <field name="tax_group_id" ref="account.tax_group_taxes"/>
+    </record>
+
+    <record id="ri_tax_other_taxes_compras" model="account.tax">
```

Comment:
???? why account.tax and not account.tax.template?

Reply:
@jco-odoo  In this case we do not want to create a whole template because is not really data, the templates we have right now are data, and then we have then change in demo just for demo purpose.

In this case this account.tax are only demo data we want to leave reflected in demo environments to different cases in the vat book. 

If you prefer and to avoid any confusion, we can add this account.tax demo in a separate demo file?